### PR TITLE
Created "Manage model inventory custom fields" page in Guides

### PR DESCRIPTION
**Pull request by validbeck**
## Internal Notes for Reviewers

> For sc-4186, there was a gap in our documentation for configuring custom fields before rich text editor templates could be referenced, so we created "Manage model inventory custom fields" page under `Guides`.

### Manage model inventory custom fields
- Net new page
- Hyperlink in sidebar is shortened, should say: `Manage inventory custom fields` 
Screenshot 2024-05-23 at 10 50 58 AM

### Guides:
Added the new `Manage model inventory custom fields` page to the listings:
Screenshot 2024-05-23 at 10 51 09 AM

### CODE REVIEW

**Comment by validbeck** [comment]
@nrichers Just a quick change to the page to add a `Delete custom fields` section as Rod confirmed that's possible.

> See attached PDF for new page: [Manage model inventory custom fields.pdf](https://github.com/validmind/documentation/files/15421600/Manage.model.inventory.custom.fields.pdf)

File: site/guide/manage-inventory-custom-fields.qmd
Comment (nrichers) [inline site/guide/manage-inventory-custom-fields.qmd:23]:
```suggestion
    At a minimum, you must provide the required information indicated by `*`. You can also choose to indicate that a field is **required on registration**.
```

FYI, weaving references to UI elements like **Required on registration** works in English text, but can break future translation. It might be easier to just make this plain text, bolded for emphasis.

File: site/guide/manage-inventory-custom-fields.qmd
Comment (nrichers) [inline site/guide/manage-inventory-custom-fields.qmd:52]:
```suggestion
2. Click [**Settings**]({{< var vm_url >}}/settings).
```

Same rational as previous suggested change — use consistent, simplified English.

File: site/guide/manage-inventory-custom-fields.qmd
Comment (nrichers) [inline site/guide/manage-inventory-custom-fields.qmd:54]:
```suggestion
3. Under **Workspace Settings**, click on **Custom Fields**. 
```

File: site/guide/manage-inventory-custom-fields.qmd
Comment (nrichers) [inline site/guide/manage-inventory-custom-fields.qmd:56]:
```suggestion
4. Select **{{< fa pencil >}} Edit** under the **Actions** column for the field you'd like to delete.
```

UI references should be bolded.

File: site/guide/manage-inventory-custom-fields.qmd
Comment (nrichers) [inline site/guide/manage-inventory-custom-fields.qmd:60]:
```suggestion
    After you confirm the deletion, the field is removed.
```

Rather than explaining the behavior of the UI, explain the user action.

**Review by nrichers — APPROVED** [review]
LGTM after comments are addressed. A few suggestions relate to tech writing best practices — e.g. explaining the user action rather than UI behavior — let me know if you want to chat more about these.

File: site/guide/manage-inventory-custom-fields.qmd
Comment (validbeck) [reply site/guide/manage-inventory-custom-fields.qmd:56]: I thought this was only for things you could click on? (As per your original style guide that I cannibalized from!)

Screenshot 2024-05-23 at 1 39 34 PM

**Comment by validbeck** [comment]
@nrichers I adjusted these changes locally then pushed them instead as I had to accommodate for Rod's quick work on PR-[#771](https://github.com/validmind/frontend/pull/771) as well!

**Comment by nrichers** [comment]
Nice 

> @nrichers I adjusted these changes locally then pushed them instead as I had to accommodate for Rod's quick work on PR-[#771](https://github.com/validmind/frontend/pull/771) as well!

I plan to publish the docs site later today with the release notes. I'll wait for this PR to get merged.

File: site/guide/manage-inventory-custom-fields.qmd
Comment (nrichers) [reply site/guide/manage-inventory-custom-fields.qmd:56]: Probably a semantic quibble, but I'm going to say that interacting includes reading.

**Comment by nrichers** [comment]
FYI, copying changes manually and force pushing should not be necessary — just do a `git pull` on your working branch to get the latest commits from GitHub.

File: site/guide/manage-inventory-custom-fields.qmd
Comment (validbeck) [reply site/guide/manage-inventory-custom-fields.qmd:56]: Hahaha, I see your argument but I think in this case the two bolds defeat the purpose. If we go with reading is interacting, then EVERYTHING should be bolded... 😆

**Comment by nrichers** [comment]
> Oh yeah, I had just made changes before I saw your contributions so I already had a divergence 💀

One of the hallmarks of git & GitHub is decentralized collaboration, so resolving divergence is as simple as merging in changes. It's literally what the thing was made for. 

Great talk by Linus Torvalds on the motivation behind git: https://youtu.be/idLyobOhtO4?si=3pZvZe9LKjTJgSlA

**Pull request merged**
